satellite/console/auth.go

```diff
@@ -66,7 +66,7 @@ func GetAuth(ctx context.Context) (Authorization, error) {
 	}
 
 	if _, ok := value.(error); ok {
-		return Authorization{}, errs.New(internalErrMsg)
+		return Authorization{}, ErrConsoleInternal.New("")
```

Comment:
empty message? what would be logged in this case?

Reply:
Internal Error:

Reply:
previous implementation didn't provide any detailed information.
return point ``InternalErrMsg``

Reply:
added more details